speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/CMakeLists.txt

```diff
@@ -59,14 +59,14 @@ SET(_HEADERS
 )
 
 ADD_INTERFACE_INCLUDEDIR()
-ADD_SDLIB_INCLUDEDIR(tgf math portability frontend rppUtils)
+ADD_SDLIB_INCLUDEDIR(tgf math portability frontend rppUtils unitTestUtilities)
 
 SD_ADD_LIBRARY(backend STATIC ${_SOURCES} ${_HEADERS} ${_INLINES})
 SET_TARGET_PROPERTIES(backend PROPERTIES LINKER_LANGUAGE CXX)
 
 TARGET_LINK_LIBRARIES(backend frontend)
 
-ADD_SDLIB_LIBRARY(backend tgf portability)
+ADD_SDLIB_LIBRARY(backend tgf portability unitTestUtilities)
```

Comment:
Using unitTestUtiltities in non unit tests, should move the utilities to RPPUtils or another file

Reply:
not possible as they need to be build, I can only move them to backend if thats what you want

Reply:
But I am only using something from the unitTestUtilities as they allow me to create runtime unit tests for the bb, so its not even that weird to do this